cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClientTest.java

```diff
@@ -219,4 +219,17 @@ public void testKeyVaultAccessPolicyListForServicePrincipal() {
         Assert.assertFalse(underTest.checkKeyVaultAccessPolicyListForServicePrincipal(accessPolicies, "dummy"));
     }
 
+    @Test
+    void getVaultNameFromEncryptionKeyUrlTestPass() {
+        String vaultName = underTest.getVaultNameFromEncryptionKeyUrl("https://dummyvaultName.vault.azure.net/keys/dummykeyName/dummykeyVersion");
+        assertThat(vaultName).isNotNull();
+        assertThat(vaultName).isEqualTo("dummyvaultName");
```

Comment:
This is a little bit redundant. 😄 The second implies the first one, but I do not mind keeping it.